common/ledger/blockledger/fileledger/factory_test.go

```diff
@@ -222,7 +222,7 @@ func TestRemove(t *testing.T) {
 		err = f.Remove("foo")
 		require.EqualError(t, err, "oogie")
 
-		dest := filepath.Join(dir, "filerepo", "remove", "foo.remove")
+		dest := filepath.Join(dir, "pendingops", "remove", "foo.remove")
 		_, err = os.Stat(dest)
 		require.NoError(t, err, "Expected foo.remove to exist")
 	})
```

Comment:
Missed the rename in the test case right below this one.

Reply:
Thanks for catching that

Reply:
Fixed